src/enway_reach_rs_driver/parser.py

```diff
@@ -71,11 +71,12 @@ def convert_time(nmea_utc):
         hours = int(nmea_utc[0:2])
         minutes = int(nmea_utc[2:4])
         seconds = int(nmea_utc[4:6])
+        seconds_tenths = float(nmea_utc[6:8]) if '.' in nmea_utc else 0
```

Comment:
Why take only tenths and not the full 2 digits of the fraction of a second?

E.g. with

```
        seconds_tenths = float(nmea_utc[6:]) if '.' in nmea_utc else 0
```